packages/serverless-orchestration/test/ServerlessHub.js

```diff
@@ -396,24 +396,24 @@ describe("ServerlessHub.js", function () {
           MONITOR_CONFIG: { contractVersion: "2.0.1", contractType: "ExpiringMultiParty" },
         },
       },
-      testServerlessLiquidator: {
-        serverlessCommand: "yarn --silent liquidator --network test",
+      testServerlessMonitor2: {
```

Comment:
What is this change? Was this meant to go into this PR?

Reply:
this was unrelated...sorry I was working on multiple PRs at the same time and this was changed due to this PR: https://github.com/UMAprotocol/protocol/pull/4309

I've reverted